atm.js

```diff
@@ -1,21 +1,45 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
+var checkingWithdrawButton = $("#withdrawButton");
+var checkingBalance = $("#checkingBalance");
+var checkingNewDeposit = $("#valueinput");
+var checkingValue = $("#checkingNewDeposit.val()");
+var cBalance= checkingBalance.html().substr(1);
+var checkingDespositButton = $("#depositButton");
+checkingDespositButton.on("click", function(){
+  var checkingNewDeposit = $("#valueinput");
+  var checkingTotal = parseInt(checkingValue) + parseInt(cBalance);
+  var cBalance= checkingBalance.html().substr(1);
+    console.log(checkingTotal)
```

Comment:
excellent job on indentation on this assignment! there are just a few `console.log`s that should be aligned with the rest of the code.